scripts/zones/Northern_San_dOria/npcs/Ailbeche.lua

```diff
@@ -15,9 +15,9 @@ local entity = {}
 
```

Comment:
This lua needs the following library:
require("scripts/globals/npc_util")

Reply:
WILCO